src/utils/createFetchOptions.js

```diff
@@ -2,10 +2,10 @@ import createRequestHeaders from './createRequestHeaders';
 import applyPlugins from './applyPlugins';
 
 export default async ({ configuration, parameters } = {}) => {
-  const { getAccessToken, plugins: { request: requestPlugins = [] } = {}, version } = configuration;
+  const { getAccessToken, plugins: { request: requestPlugins = [] } = {}, version, customHeaders = {} } = configuration;
```

Comment:
Don't forget to update the documentation as well 😄

Reply:
Also I would call this `additionalHeaders`, like we do for `additionalActions`

Reply:
Alright, will rename this 👍